src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/OffenderService.kt

```diff
@@ -423,7 +424,11 @@ class OffenderService(
     }
   }
 
-  fun getPrisonCaseNotesByNomsNumber(nomsNumber: String): AuthorisableActionResult<List<CaseNote>> {
+  fun getFilteredPrisonCaseNotesByNomsNumber(nomsNumber: String, getCas1SpecificNoteTypes: Boolean): CasResult<List<CaseNote>> {
+    val cas1PrisonNoteTypes = listOf(
```

Comment:
for clarity could we please rename this to `cas1PrisonNoteTypesToInclude`. I don't know why but my brain assumed it was notes to exclude for a few minutes and I got confused :)